runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -820,7 +840,7 @@ NOTE: you can use `y`{normal} as an operator: `yw`{normal} yanks one word.
 ~~~ cmd
         :set noic
 ~~~
- 7. To toggle the value of a setting, prepend it with "inv":
+ 7. To invert the value of a setting, prepend it with "inv":
```

Comment:
Not sure which of these terms is more widely understandable; personally, I'm more familiar with "toggle" -- but I'm hardly the target audience for this...

Reply:
This change is to match the inv prefix it is talking about, hoping it will help people remember